tests/zfs-tests/include/libtest.shlib

```diff
@@ -20,14 +20,13 @@
 #
 
 #
-# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
-# Use is subject to license terms.
-# Copyright (c) 2012, 2018 by Delphix. All rights reserved.
-# Copyright (c) 2017 by Tim Chase. All rights reserved.
-# Copyright (c) 2017 by Nexenta Systems, Inc. All rights reserved.
-# Copyright (c) 2017 Lawrence Livermore National Security, LLC.
-# Copyright (c) 2017 Datto Inc.
-# Copyright (c) 2017 Open-E, Inc. All Rights Reserved.
+# Copyright (c) 2009, Sun Microsystems Inc. All rights reserved.
+# Copyright (c) 2012, 2018, Delphix. All rights reserved.
+# Copyright (c) 2017, Tim Chase. All rights reserved.
+# Copyright (c) 2017, Nexenta Systems Inc. All rights reserved.
+# Copyright (c) 2017, Lawrence Livermore National Security LLC. All rights reserved.
```

Comment:
nit: This line breaks the 79 character limit.  Let's simply remove the duplicate line here and call it a day or drop this hunk entirely.

Reply:
I think c style skips commenting...

Reply:
@behlendorf I think you mean "All rights reserved" with "the duplicate line"?

Reply:
Removed "All rights reserved'.
Needs squash later (not at main workstation atm)